src/main/java/seedu/mindmymoney/constants/PrintStrings.java

```diff
@@ -0,0 +1,12 @@
+package seedu.mindmymoney.constants;
+
+/**
+ * Container for indexes used by handlers.
+ */
+public class PrintStrings {
+    private PrintStrings() {
+    }
+
+    public static final String LINE = "---------------------------------------" +
```

Comment:
I don't really like having the lines as it makes the display abit cluttered? I was going for abit of minimalist design, but can just leave it here first and see what the rest feel about it.

Reply:
i see, this one can discuss when we meet